src/main/resources/lookml/schema-schema.lkml

```diff
@@ -0,0 +1,106 @@
+# Licensed to the LookML Authors under one or more contributor
+# license agreements.  See the NOTICE file distributed with this
+# work for additional information regarding copyright ownership.
+# The LookML Authors license this file to you under the Apache
+# License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License.  You may obtain a
+# copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied.  See the License for the specific
+# language governing permissions and limitations under the
+# License.
+#
+# Definition of the schema of a Schema-LookML file.
+#
+# This is the 'meta schema', and is self-defining.
+#
+schema: schema {
```

Comment:
if these are named schemas, should we consider a specific name?  ie..  `schema: lookml_schema_v1` or something?

Reply:
No, the versioning is handled orthogonally (e.g. by a source control system, or by including a directory that contains the version number). This code base should not support more than one version of the schema schema at the same time.